tools/docs/manager.ts

```diff
@@ -113,19 +113,36 @@ export async function getManagersGitHubIssues(): Promise<
 export async function generateManagers(dist: string): Promise<void> {
   const managers = getManagers();
   const managerIssuesMap = await getManagersGitHubIssues();
-  const allLanguages: Record<string, string[]> = {};
```

Comment:
shouldn't we leave languages until we remove them completely? 🤔

Reply:
I think of this as a "soft" deprecation, as every programming language has an same named category.